specs/order_spec.rb

```diff
@@ -79,32 +81,59 @@
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
-    it "Returns an array of all orders" do
-      # TODO: Your test code here!
+    xit "Returns an array of all orders" do
+      #act
+      results = Grocery::Order.all
+      # print results
+      #assert
+      results.must_be_kind_of Array
+      results[0].must_be_kind_of Grocery::Order
     end
 
-    it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
+    xit "Returns accurate information about the first order" do
+      # Arrange - feed information for method to work with
+
+      products_list = ["Slivered Almonds", "22.88"], ["Wholewheat flour", "1.93"],["Grape Seed Oil" , "74.9"]
+
```

Comment:
The product lists in orders you get from `.all` should have the same form as the hashes we saw in previous tests.